wiki/Announcements/Contests/en.md

```diff
@@ -5,7 +5,9 @@
 
 # Contests
 
-## Monthly Beatmapping Contest
+## Official Beatmapping Contests
+
```

Comment:
"All sections must contain some content, even those that act like a container for subsections." -ASG